settings/common.py

```diff
@@ -16,7 +16,9 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = utils.get_bool_environ("DEBUG", False)
 
-ALLOWED_HOSTS = utils.get_list_environ("ALLOWED_HOSTS", "localhost,127.0.0.1")
+ALLOWED_HOSTS = utils.get_list_environ(
```

Comment:
We should look into a more elegant way of switching the allowed hosts depending on the environment context. (This would do for now)

Reply:
What do you mean when you say a more elegant way?

Reply:
like perhaps including localhost only if running in debug mode? since i assume localhost is something you don't want open in production